__tests__/lint/info-contact--deprecated-format/snapshot.js

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E lint info-contact--deprecated-format 1`] = `
+The apiDefinitions field is deprecated. Use apis instead.
```

Comment:
We need to hide it until we publish docs
```suggestion
The apiDefinitions field is deprecated. Use apis instead, see changelog: <link here>
```